core/templates/dev/head/error/disabled_exploration.html

```diff
@@ -3,6 +3,7 @@
 {% block navbar_breadcrumb %}
 {% endblock navbar_breadcrumb %}
 
+<!-- TODO(mili) Translate the title. To avoid FOUC it must be done in the js -->
```

Comment:
It's a little bit complicated. On one hand, this page and the error page are rendered from the python controllers and use only jinja. This means there is no particular javascript file that loads only with this page where to put $rootScope.title = someTranslation. For each page where we want to change the title, we should add a controller.
On the other hand, there is no way (that I know) of replace only one part of the title to translate only {% block subtitle %} and leave {% block title %} as it was.
One solution is:
- Remove the subtitle block and leave only the title.
- In each html page, fill the title block with the corresponding translation id.
- In the base.js script add a function that reads the translation id from the html using $window.document.title and replaces it with the corresponding translation. (Using the same code we are using now to change the gallery title, for example.)
Let me know what you think!